packages/flutter/lib/src/material/text_field.dart

```diff
@@ -317,6 +317,7 @@ class TextField extends StatefulWidget {
     this.canRequestFocus = true,
     this.spellCheckConfiguration,
     this.magnifierConfiguration,
+    this.canTapOutsideFocus = true,
```

Comment:
This should probably also be added to `CupertinoTextField`, and `TextFormField`.